README.md

```diff
@@ -27,52 +27,52 @@ struct PlayerResolver: Resolver {
 }
 ```
 
-This is a trivial example which on its own may not warrant a special "Resolver," but as we'll see below in the "Who is Settler For?" section, a Resolver can be particularly helpful in the development and maintenance of _large_, _complex_, and _interconnected_ dependency graphs.
+This is a trivial example which on its own may not warrant a special “resolver,” but as we’ll see below in the “Who is Settler for?” section, a resolver can be helpful in the development and maintenance of large and dense dependency graphs.
 
-See [Resolver.md](https://github.com/daltonclaybrook/Settler/blob/main/Resolver.md) for an in-depth Resolver implementation guide.
+See [Resolver.md](https://github.com/daltonclaybrook/Settler/blob/main/Resolver.md) for an in-depth `Resolver` implementation guide.
 
-See the [SettlerDemo directory](https://github.com/daltonclaybrook/Settler/tree/main/Sources/SettlerDemo) for a more detailed example of a Resolver.
+See the [SettlerDemo directory](https://github.com/daltonclaybrook/Settler/tree/main/Sources/SettlerDemo) for a more detailed resolver example.
 
 ## Compiler magic
 
-The power (and magic ✨) of Settler lies in its ability to parse your Resolvers in-line with the Swift compiler and report errors directly in Xcode as if it were part of the compiler itself.
+The power (and magic ✨) of Settler lies in its ability to parse your resolvers alongside the Swift compiler and report errors directly in Xcode as if it were part of the toolchain itself.
 
-By defining each of your dependencies as a function, and by using `Key` members as inputs/outputs of those functions, Settler is able to resolve your entire dependency graph in the correct order, ignoring types that are unused, lazily initializing dependencies where necessary, and reporting errors when things are wrong or missing.
+By defining your dependencies as a function, and by using `Key` members as inputs and outputs of those functions, Settler is able to resolve your dependency graph in the correct order, ignoring types that are unused, lazily initializing dependencies where necessary, and reporting errors when things aren’t quite right.
```

Comment:
>By defining your dependencies as a function

Does this sound like you're defining all your dependencies as just one function? Should this be, "By defining your dependencies as functions..."?